api/handlers_buckets.go

```diff
@@ -89,7 +89,7 @@ func (s *server) BucketCreateHandler(w http.ResponseWriter, r *http.Request) {
 	rollBackTasks = append(rollBackTasks, rbfunc)
 
 	// wait for the bucket to exist
-	err = retry(5, 2*time.Second, func() error {
+	err = retry(3, 2*time.Second, func() error {
```

Comment:
These retry numbers are confusing, since it's doing some exponential delay, but I'm not clear how much. I think this times out after 20 something seconds, instead of 6

Reply:
it uses a random amount of jitter actually, at most each try will sleep twice as long.